edx_exams/apps/core/admin.py

```diff
@@ -70,9 +70,9 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
 @admin.register(CourseExamConfiguration)
 class CourseExamConfigurationAdmin(admin.ModelAdmin):
     """ Admin configuration for the Course Exam Configuration model """
-    list_display = ('course_id', 'provider', 'allow_opt_out')
-    readonly_fields = ('course_id', 'provider')
-    search_fields = ('course_id', 'provider__name', 'allow_opt_out')
+    list_display = ('course_id', 'provider', 'allow_opt_out', 'escalation_email')
+    readonly_fields = ('course_id', 'provider', 'escalation_email')
```

Comment:
since you can't do any real harm in editing this field it might make sense to allow that. I agree writing up extra validation does not seem worthwhile. If the validation is just on the serializer, maybe we don't need it in admin? It looks like there's no integrity problem or errors created if a staff user does set this to blank. (for whatever reason they feel is necessary)